### DISCUSSION TITLE
Τεκμηρίωση αναφοράς ΓΕ με εικόνες

### DISCUSSION THREAD

**Discussion by gregtsag** [sdy60/syllabus#28] [Q&A]
Καλησπέρα,

Για τις ενδιεικτικές εικόνες αρκεί το link του project ή θέλετε να ανεβάσουμε και εικόνες ή σχόλια;

**Comment by epidrome** [comment, marked as answer]
Η αναφορά της ΓΕ θα πρέπει να είναι όσο γίνεται περισσότερο αυτοτελής, όπως περιγράφεται στα εισαγωγικά στοιχεία της ΓΕ

Αν εξαιρέσουμε μεγάλα αρχεία βίντεο και κώδικα, οι εικόνες πρέπει να είναι μέρος της αναφοράς
